yoga/YGEnumsPrint.h

```diff
@@ -0,0 +1,359 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#pragma once
+
+#include "YGMacros.h"
+#include "Yoga.h"
+
+YG_EXTERN_C_BEGIN
+
+void YGLogAlign(YGLogLevel logLevel, const char * param, YGAlign value){
```

Comment:
File is names `YGEnumsPrint` but functions are named `YGLogX`. The mix of `Print` and `Log` is a bit odd.